app/views/estimates/_capital_table.html.slim

```diff
@@ -32,10 +32,10 @@
         - vehicle.each do |key, value|
           - body.with_row do |row|
             - row.with_cell(header: true, text: t("estimates.show.vehicle.#{key}"))
-            - row.with_cell(text: value, numeric: true)
+            - row.with_cell(text: value, html_attributes: { class: "govuk-!-text-align-right" })
```

Comment:
Ok, so basically the cells with numbers in, by removing `numeric: true` we're causing them _not_ to have the class `govuk-table__cell--numeric` applied, and instead we apply `govuk-!-text-align-right`. I believe you that that works, but I don't know _why_ it would work. I guess the original class applies `font-variant-numeric: tabular-nums` and `font-feature-settings: "tnum" 1;`. Those are unusual - could they be having an odd interaction with puppeteer when generating the PDF?

We probably don't _need_ to know the answer, but at the very least we should probably document somewhere why we're making this change. As a comment, or in the readme. Or maybe best to create a reusable method called `def pdf_friendly_numeric_table_cell(row, value, additional_classes = [])` and call that every time we want to put a number in a table cell?

Reply:
Sorry @patrick-laa not quite following the reusable method. 

- Placed this in the `def pdf_friendly_numeric_table_cell(row, value, additional_classes = []) end` in the _app/helpers/results_helper.rb_ file.


- Calling it like so:
```
- body.with_row do |row|
  ruby:
    text = header_with_hint("estimates.show.client_income_types.#{income_type}", "estimates.show.client_income_hints.#{income_type}")
  - row.with_cell(header: true, text:)
  - pdf_friendly_numeric_table_cell(row.with_cell, text: money_amount, html_attributes: { class: "govuk-!-text-align-right" })

```

Reply:
@MazOneTwoOne so I'm thinking it would be:

```slim
- pdf_friendly_numeric_table_cell(row, money_amount)
```

So the thing that calls the method doesn't need to know anything about CSS classes or what specifically it has to _do_ to make it pdf-friendly.

If there were other CSS classes you needed in other places you could pass them in, e.g:

```slim
- pdf_friendly_numeric_table(row, @model.household_vehicle_assessed_value, ["govuk-!-font-weight-bold"])
```